stdlib/math/math.flux

```diff
@@ -0,0 +1,198 @@
+package math
+
+// builtin constants
+builtin pi
+builtin e
+builtin phi
+builtin sqrt2
+builtin sqrte
+builtin sqrtpi
+builtin sqrtphi
+builtin ln2
+builtin log2e
+builtin ln10
+builtin log10e
+builtin maxfloat
+builtin maxint
+builtin minint
+builtin maxuint
+
+//missing builtin values [y1 sqrte roundtoeven nextafter32 sqrtphi sqrtpi copysign ceil], extra builtin values [Ceil sqrtPhi sqrtPi sqrtE roundToEven copySign]
+// builtin functions
+builtin abs
+builtin acos
+builtin acosh
+builtin asin
+builtin asinh
+builtin atan
+builtin atan2
+builtin atanh
+builtin cbrt
+builtin ceil
+builtin copysign
+builtin cos
+builtin cosh
+builtin dim
+builtin erf
+builtin erfc
+builtin erfcinv
+builtin erfinv
+builtin exp
+builtin exp2
+builtin expm1
+builtin float64bits
+builtin float64frombits
+builtin floor
+builtin frexp
+builtin gamma
+builtin hypot
+builtin ilogb
+builtin m_inf
+builtin isInf
+builtin isNaN
+builtin j0
+builtin j1
+builtin jn
+builtin ldexp
+builtin lgamma
+builtin log
+builtin log10
+builtin log1p
+builtin log2
+builtin logb
+builtin m_max
+builtin m_min
+builtin mod
+builtin modf
+builtin NaN
+builtin nextafter
+builtin pow
+builtin pow10
+builtin remainder
+builtin round
+builtin roundtoeven
+builtin signbit
+builtin sin
+builtin sincos
+builtin sinh
+builtin sqrt
+builtin tan
+builtin tanh
+builtin trunc
+builtin y0
+builtin y1
+builtin yn
+
+// Table transformations for trig functions
+sinValue = (tables=<-) => tables |> map(fn:(r) => sin(x:r._value))
+cosValue = (tables=<-) => tables |> map(fn:(r) => cos(x:r._value))
+tanValue = (tables=<-) => tables |> map(fn:(r) => tan(x:r._value))
+asinValue = (tables=<-) => tables |> map(fn:(r) => asin(x:r._value))
+acosValue = (tables=<-) => tables |> map(fn:(r) => acos(x:r._value))
+atanValue = (tables=<-) => tables |> map(fn:(r) => atan(x:r._value))
```

Comment:
I don't think we want these functions, they provide little value in my opinion. Thoughts?

Reply:
Yes, I agree. Are these function here because it is supposed that trig functions are commonly used like this?  
Maybe we could provide a function generator in the prelude:

```
valueMapperFor = (func) => {
    return (tables=<-) => tables |> map(fn:(r) => func(r._value))
}
```

To avoid verbose syntax to users when they want to write those functions. However, it would require positional arguments...

Reply:
yeah, they should not be permanent.  I have them in the demo to show that it's something you can do, will remove before I merge.